assets/js/app.js

```diff
@@ -22,6 +22,9 @@ import UserForm from "./user_form";
 import VegaLite from "./vega_lite";
 import morphdomCallbacks from "./morphdom_callbacks";
 import { loadUserData } from "./lib/user";
+import html2canvas from "html2canvas";
+
+window.html2canvas = html2canvas;
```

Comment:
You should be able to import `html2canvas` directly in `session/index.js` and use it without setting globally.

Reply:
Nice one, this makes sense.

Reply:
Regarding the padding, I have not found a solution for that as of yet.